invenio_app_ils/ui/backoffice/src/pages/ItemDetails/components/ItemPendingLoans/state/__tests__/selectors.js

```diff
@@ -0,0 +1,57 @@
+import { serializePendingLoan } from '../selectors';
+import { fromBackend } from 'common/api/date';
+
+const stringDate = '2018-01-01T11:05:00+01:00';
+
+describe('Pending loans serialization tests', () => {
+  it('should serialize only id and updated if metadata empty', () => {
+    const serialized = serializePendingLoan({
+      id: '123',
+      updated: stringDate,
+      metadata: {},
+    });
+
+    expect(serialized).toEqual({
+      loan_pid: '123',
+      updated: fromBackend(stringDate),
```

Comment:
`fromBackend ` do not forget to rename this one if you go with my proposal.

Reply:
Done!